File: app/src/embed/assets/toybox/entities/reg.py
def main():
    registered_username = input("Enter username for registration: ")
    registered_password = input("Enter password for registration: ")

    print("Registration successful!")

    logged_in = False
    while not logged_in:
        username = input("Enter username for login: ")
        password = input("Enter password for login: ")

        if username == registered_username and password == registered_password:
            print(f"Login successful! Welcome, {username}")
            logged_in = True
        else:
            print("Invalid username or password. Try again.")


main()

File: app/src/embed/assets/toybox/entities/reg.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter username for registration: ");
        String registeredUsername = scanner.nextLine();

        System.out.print("Enter password for registration: ");
        String registeredPassword = scanner.nextLine();

        System.out.println("Registration successful!");

        boolean isLoggedIn = false;
        while (!isLoggedIn) {
            System.out.print("Enter username for login: ");
            String username = scanner.nextLine();

            System.out.print("Enter password for login: ");
            String password = scanner.nextLine();

            if (username.equals(registeredUsername) && password.equals(registeredPassword)) {
                System.out.println("Login successful! Welcome, " + username);
                isLoggedIn = true;
            } else {
                System.out.println("Invalid username or password. Try again.");
            }
        }

        scanner.close();
    }
}